src/pages/edge/adobe-journey-optimizer-decisioning/tabs/api-reference.md

```diff
@@ -444,7 +631,7 @@ AEPDecisionScope* decisionScope2 = [[AEPDecisionScope alloc] initWithName: @"myS
                               andData: @{@"dataKey": @"dataValue"}];
 ```
 
-<Variant platform="ios" api="update-propositions-withError" repeat="12"/>
```

Comment:
Same comment as Ashu made. please check if this is referenced from '/adobe-journey-optimizer-decisioning/api-reference.md', please rename there if linking is broken.

Reply:
done @spoorthipujariadobe